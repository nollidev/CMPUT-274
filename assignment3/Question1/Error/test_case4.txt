Traceback (most recent call last):
  File "/home/nattyeddy/Documents/School/Uni Homework/Second Year/CMPUT-274/assignment3/Question1/solutionQuestion1.py", line 35, in <module>
    main()
    ~~~~^^
  File "/home/nattyeddy/Documents/School/Uni Homework/Second Year/CMPUT-274/assignment3/Question1/solutionQuestion1.py", line 31, in main
    read_tree().print_tree()
    ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'print_tree'
