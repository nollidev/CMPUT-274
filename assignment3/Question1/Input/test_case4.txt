24 3
W A B C D E - F G X H I - J - - - - - - - - K L
