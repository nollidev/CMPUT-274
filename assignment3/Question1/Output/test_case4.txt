['W', 'A', 'B', 'C', 'D', 'E', '-', 'F', 'G', 'X', 'H', 'I', '-', 'J', '-', '-', '-', '-', '-', '-', '-', '-', 'K', 'L']
