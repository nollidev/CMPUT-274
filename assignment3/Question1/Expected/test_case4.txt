W -> A B C
A -> D E
D -> J
B -> F G X
F -> K L
C -> H I
